provider/oauth2/forms.py

```diff
@@ -340,10 +340,15 @@ def clean(self):
 
 class ClientCredentialsGrantForm(ScopeMixin, OAuthForm):
     """ Validate a client credentials grant request. """
+    scope = forms.CharField(required=False)
 
-    def clean(self):
-        cleaned_data = super(ClientCredentialsGrantForm, self).clean()
-        # We do not fully support scopes for this grant type; however, a scope is required
-        # in order to create an access token. Default to read-only access.
-        cleaned_data['scope'] = constants.READ
-        return cleaned_data
+    def clean_scope(self):
+        # NOTE (CCB): This is a horrible hack, like much of our OAuth work. The scopes are declared in
```

Comment:
Is this a temporary hack that will be cleaned up once we migrate completely to DOT?